Doc/library/csv.rst

```diff
@@ -95,11 +95,11 @@ The :mod:`csv` module defines the following functions:
    :func:`list_dialects` function.  The other optional *fmtparams* keyword arguments
    can be given to override individual formatting parameters in the current
    dialect.  For full details about dialects and formatting parameters, see
-   the :ref:`csv-fmt-params` section. To make it
+   the :ref:`csv-fmt-params` section.  To make it
```

Comment:
Why the second space is required? For the record, I've seen it in various manuals and license texts and headers.

Reply:
All the page uses two spaces after periods, so for consistency.

Reply:
HTML rendering collapses whitespace, so these changes have no effect.
One could argue that it makes it easier to read and edit the source, but that point is debated.
So in the absence of consensus, I would not change these.